project/Dependencies.scala

```diff
@@ -24,7 +24,7 @@ object Dependencies {
   lazy val scalaCheckVersion = settingKey[String]("The version of ScalaCheck to use.")
 
   val Versions = Seq(
-    crossScalaVersions := Seq("2.12.10", "2.11.12", "2.13.1"), // also update .travis.yml when changing here to avoid confusion
+    crossScalaVersions := Seq("2.12.15", "2.11.12", "2.13.1"), // also update .travis.yml when changing here to avoid confusion
```

Comment:
I also want to update to Scala 2.13.7 but that is a bit more difficult and I removed it from here to unblock other backports.